src/meta/MetaServiceUtils.cpp

```diff
@@ -185,6 +199,26 @@ std::string MetaServiceUtils::schemaTagKey(GraphSpaceID spaceId, TagID tagId, Sc
     return key;
 }
 
+std::string MetaServiceUtils::schemaTagVal(const std::string& name, nebula::cpp2::Schema schema) {
```

Comment:
Why move the code?

Reply:
I wish to put the same set of methods together.